printers.py

```diff
@@ -29,16 +29,18 @@ def print_subscription_status(update, context):
 
     if subscription:
         subscription_limit = subscription.kwargs['created_at'] + datetime.timedelta(days=7)
-        date_object = subscription_limit.strftime("%d-%m-%Y %H:%M:%S")
+        subscription_limit_date_time = subscription_limit.strftime("%d-%m-%Y %H:%M:%S")
+        deadline = subscription.kwargs['deadline'].strftime("%d-%m-%Y")
 
         department = Buro.get_buro_by_id(subscription.kwargs['buro'])
         msg.reply_text(
-            'Current subscription details:\n\n - Department: %s \n - Type: %s \n - Interval: %s \n - Until: %s \n' % (
-                department.get_name(), subscription.kwargs['termin'], subscription.trigger.interval, date_object))
+            'Current subscription details:\n\n - Department: %s \n - Type: %s \n - Interval: %s \n - Until: %s \n - Deadline: %s \n' % (
+                department.get_name(), subscription.kwargs['termin'], subscription.trigger.interval,
+                subscription_limit_date_time, deadline))
         print_unsubscribe_button(chat_id)
 
 
-def notify_about_termins(chat_id, buro, termin, created_at):
+def notify_about_termins(chat_id, buro, termin, created_at, deadline):
```

Comment:
Please make deadline optional to keep existing subscriptions working

Reply:
Done :+1: